Makefile

```diff
@@ -113,6 +113,10 @@ crystal: $(O)/crystal ## Build the compiler
 deps: $(DEPS) ## Build dependencies
 llvm_ext: $(LLVM_EXT_OBJ)
 
+.PHONY: format
+format: ## Format sources
+	./bin/crystal tool format$(if $(check), --check) src spec samples
+
```

Comment:
Would you mind adding this to `Makefile.win` as well? And can `check` be documented among the variables above?